ddtrace/contrib/botocore/patch.py

```diff
@@ -319,14 +322,24 @@ def patch():
 
     wrapt.wrap_function_wrapper("botocore.client", "BaseClient._make_api_call", patched_api_call)
     Pin(service="aws").onto(botocore.client.BaseClient)
+    _PATCHED_SUBMODULES.clear()
 
 
 def unpatch():
+    _PATCHED_SUBMODULES.clear()
     if getattr(botocore.client, "_datadog_patch", False):
         setattr(botocore.client, "_datadog_patch", False)
         unwrap(botocore.client.BaseClient, "_make_api_call")
 
 
+def patch_submodules(submodules):
```

Comment:
```suggestion
def patch_submodules(submodules):
    # type: (Union[List[str], bool]) -> None
```

Correct? (if you accept, we'll need a `from typing import Union` above.